docs/install.rst

```diff
@@ -272,6 +272,16 @@ will look like this.
     PS> ./bin/crate
 
 
+Notes about Linux and macOS
+---------------------------
+
+You can also install and run CrateDB on Linux and macOS with one simple command
+in your terminal application:
```

Comment:
```suggestion
If you want to try out CrateDB but would prefer to avoid the hassle of
installation, you can get a fresh CrateDB node up and running in your current
working directory with a single command:
```

how about this?

- demonstrates why you would want to use this
- highlights that this isn't a "real" installation
- makes it a little more clear that this meant to be a temporary node

Reply:
Thank you!